src/AWS.Deploy.CLI/ServerMode/Models/OptionSettingItemSummary.cs

```diff
@@ -3,13 +3,16 @@
 
 using System;
 using System.Collections.Generic;
+using AWS.Deploy.Common.Recipes;
 
 namespace AWS.Deploy.CLI.ServerMode.Models
 {
     public class OptionSettingItemSummary
     {
         public string Id { get; set; }
 
+        public string FullyQualifiedId { get; set; }
```

Comment:
Could you document this new property? Unsure how it's different from `Id`?

Given other Toolkits besides Visual Studio could potentially wrap server mode in the future, think it'd be nice to document as much as possible while we're here.

Reply:
Done. I've updated the entire class based on OptionSettingItem